bears/general/ShebangBear.py

```diff
@@ -0,0 +1,32 @@
+from coalib.bears.LocalBear import LocalBear
+from coalib.results.Diff import Diff
+from coalib.results.Result import Result
+
+
+class ShebangBear(LocalBear):
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_FIX = {'Syntax'}
+
+    def run(self, filename, file):
+        """
+        Ensure that the file uses the generic Shebang operator.
+        """
+        if len(file) == 0 or '#!/usr/bin/env' in file[0]:
+            return
+
+        diff = Diff(file)
+        message = None
+
+        if '#!' in file[0]:
+            binary = file[0].split('bin/', 1)[1].strip('\n')
```

Comment:
Can you please use [`os.path.basename`](https://docs.python.org/3/library/os.path.html#os.path.basename) instead of this? Since, it causes problems if it's `/usr/sbin/`, `/sbin/`, `/usr/local/sbin`, etc.